src/foundry/client/pixi/layers/placeables/tiles.d.ts

```diff
@@ -0,0 +1,114 @@
+import type { DropData } from "../../../data/abstract/client-document";
+
+export {};
+
+declare global {
+  /**
+   * A PlaceablesLayer designed for rendering the visual Scene for a specific vertical cross-section.
+   */
+  class TilesLayer extends PlaceablesLayer<"Tile"> {
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
+     */
+    static get instance(): Canvas["tiles"];
+
+    static override documentName: "Tile";
+
+    override options: TilesLayer.LayerOptions;
+
+    /**
+     * @defaultValue
+     * ```js
+     * foundry.utils.mergeObject(super.layerOptions, {
+     *    name: "tiles",
+     *    zIndex: 0,
+     *    controllableObjects: true,
+     *    rotatableObjects: true,
+     *    elevationSorting: true
+     * })
+     * ```
+     */
+    static override get layerOptions(): TilesLayer.LayerOptions;
+
+    override get hookName(): string;
+
+    override get hud(): TileHUD;
+
+    /**
+     * An array of Tile objects which are rendered within the objects container
+     */
+    get tiles(): Tile[];
+
+    /**
+     * Get an array of overhead Tile objects which are roofs
+     */
+    get roofs(): Tile[];
+
+    /**
+     * Determine whether to display roofs
+     */
+    get displayRoofs(): boolean;
+
+    /**
+     * A convenience reference to the tile occlusion mask on the primary canvas group.
+     */
+    get depthMask(): CachedContainer;
+
+    override controllableObjects(): Generator<Tile>;
+
+    override _activate(): void;
+
+    override _deactivate(): void;
+
+    /**
+     * Activate a sublayer of the tiles layer, which controls interactivity of placeables and release controlled objects.
+     * @param foreground - Which sublayer need to be activated? Foreground or background?
+     *                     (default: `false`)
+     */
+    protected _activateSubLayer(foreground?: boolean): void;
+
+    /**
+     * @param options - Unused
```

Comment:
I noticed this in a lot of places. Is `unused a comment from Foundry? If not just leave off the documentation entirely. Foundry only has it to document the type.

Reply:
I picked it up from pre-existing documentation - these files aren't strictly de novo due to the moves. These options on draw() are basically unused throughout the entire codebase, so I just kinda gave them `Record<string, unknown>` because that's the usual structure rather than `any`